oauthenticator/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .oauth2 import *
 from .github import *
 from .bitbucket import *
+from .mediawiki import *
```

Comment:
It should generally be `oauthenticator.github.GitHubOAuthenticator`. It is already this way for CILogon.